CMakeLists.txt

```diff
@@ -155,9 +155,6 @@ endif()
 
 set(SPDLOG_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/third_party/spdlog/include)
 
-option(BPFTIME_CUDA_ROOT "Root for CUDA installation" "")
```

Comment:
Why removing these options

Reply:
They are moved to cmake/StandardSettings.cmake